history.md

```diff
@@ -0,0 +1,88 @@
+RIT LibreCorps has been partnered intermittently with the UNICEF Office of Innovation since 2014.
+As a successful pilot, LibreCorps reached up to 24 supported faculty-driven software projects!
+
+
+
+## FOSS Minor
+
+Beginning of Fall 2014 Semester, Rochester Institute of Technology started to offer the first minor in [Free and Open Source Software and Free Culture](https://www.rit.edu/study/free-and-open-source-software-and-free-culture-minor).
+The minor was brought about by additional courses offered by the School of Interactive Games and Media ([IGM](https://www.rit.edu/computing/school-interactive-games-and-media)).
+
+
+
+## FOSS@RIT:
+
+FOSS@RIT is the name of the github organization for all things open source at Rochester Institute of Technology, however, the organization isn't very official. 
+During the time period of renaming from FOSSBOX to FOSS@MAGIC, this was what you would call the Open Source hub, though interchangable with the FOSS@MAGIC name.
+This was still a community of people with interest in free and open source software at RIT, with the initial purpose of keeping FOSSBOX alive.
+
+
+
+## FOSS@MAGIC:
+
+When [Magic Spell Studio](https://www.rit.edu/magic/magic-spell-studios) was conceptualized, this organization was made to prioritize game development, which was where open source got started on this campus.
+This organization was managed by [Andrew Phelps](https://professorandrewphelps.net/) in 2015. Since the magic building was not completed at the time, they were relocated to the innovation center, in the exact location that FOSSBOX initially used. 
+FOSS Hours were introduced, which helped increase involvement of students and other interested parties. However, shortly after, Remy DeCausemaker had left Rochester Institute of Technology to go work for Twitter, which made the open source community lose its growing momentum.
+The organization still had funding and students were still getting involved, events were being run and the organization finally was located inside of Magic. FOSS@Magic existed in this manner for quite a while, but things changed once Stephen Jacobs got converstations started with UNICEF Innovation, a Tech R&D department permanently part of the United Nations System.
+
+
+
+## OPEN@RIT:
+
+In 2020, the [Alfred P. Sloan Foundation](https://sloan.org/) provided [financial support](https://sloan.org/grant-detail/10076) for an effort led by the Rochester Institute of Technology’s Stephen Jacobs to design and launch an undergraduate-anchored program: [Open@RIT](https://openr.it/about/). 
+This program was situated just under Rochester Institute of Technologies Vice President for Research, [Ryne Raffaelle](https://www.rit.edu/directory/rprsps-ryne-raffaelle) and has been the home of open source at RIT for the last four years. 
```

Comment:
I'm noticing a few small things throughout this that feel all little off grammar wise, like "was situated" vs "is situated", or "the program now participants" in the next line that may warrant a larger scale check to make sure the whole post is okay

Reply:
Yeah, I swept through it twice and fixed quite a few typos.